devicesetup/src/main/res/layout/activity_success.xml

```diff
@@ -38,14 +38,29 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
-            android:layout_marginBottom="24dp"
+            android:layout_marginBottom="12dp"
             android:layout_marginLeft="24dp"
             android:layout_marginRight="24dp"
             android:layout_marginTop="6dp"
             android:gravity="center"
             android:text="@string/setup_success_details"/>
 
         <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="6dp"
+            android:text="@string/device_naming_label"/>
+
+        <EditText
```

Comment:
Including the design support lib is fine.  We don't want to go crazy with dependencies, but for a super-common, first-party library, there's very little reason not to include it.

As for the appearance, we already have a few other modest design differences from iOS where there's a strongly established pattern on Android.  (e.g.: the use of the FAB and per-item overflow menus on Tinker's main screen).  As the old Android design guidelines say: _Deviate with purpose._  I think that applies here, too.